src/embind/emval.js

```diff
@@ -18,7 +18,7 @@
 
 // Number of handles reserved for non-use (0) or common values w/o refcount.
 {{{ 
-  globalThis.EMVAL_RESERVED_HANDLES = 5;
+  globalThis.EMVAL_RESERVED_HANDLES = 10;
```

Comment:
Can we keep this at 5 to avoid confusion?

It only two places it is used seem like they would be fine with that.

Reply:
Done.